mmv1/third_party/terraform/utils/gkeonprem_operation.go

```diff
@@ -0,0 +1,136 @@
+package google
```

Comment:
We'll want a version guard on this file, as described in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/README.md#beta-feature

However, instead of `unless version == 'ga'` we'll want `if version == 'private'`

Reply:
Sounds good. Moved to a .erb file with the guards.